pctiler/pctiler/endpoints/item.py

```diff
@@ -30,6 +30,8 @@
     colormap_dependency=PCColorMapParams,
     reader_dependency=ReaderParams,
     router_prefix=get_settings().item_endpoint_prefix,
+    # We remove the titiler default `/map` viewer
+    add_viewer=False,
```

Comment:
in titiler 0.8 we added a `/map` viewer which is added by default with the Tiler Factories